data/microsoft

```diff
@@ -17,3 +17,5 @@ azure.com
 office.com
 xbox.com
 visualstudio.com
+
+domain:microsofttranslator.com
```

Comment:
For the record, `domain:` is the default option. It is not necessary to explicitly use it.

Reply:
Thanks for reminding